tests/testsuite/support/paths.rs

```diff
@@ -0,0 +1,108 @@
+use std::cell::Cell;
+use std::env;
+use std::fs;
+use std::io::{self, ErrorKind};
+use std::path::{Path, PathBuf};
+use std::sync::atomic::{AtomicUsize, Ordering, ATOMIC_USIZE_INIT};
+use std::sync::{Once, ONCE_INIT};
+
+static INTEGRATION_TEST_DIR: &'static str = "integration_test";
+static NEXT_ID: AtomicUsize = ATOMIC_USIZE_INIT;
+
+thread_local!(static TASK_ID: usize = NEXT_ID.fetch_add(1, Ordering::SeqCst));
```

Comment:
This is because cargo runs tests in multiple threads by default, and this ensures that each test gets its own directory. So all of the acceptance tests are sandboxed in `target/`, and these methods will create dirs `target/acceptance_test/t0`, `target/acceptance_test/t1`, etc.